### ISSUE TITLE
Schema validation update functionality

### ISSUE THREAD

**Issue by SimonasTr** [joowani/python-arango#151]
Seems that there is no way to update cchema validation rules through library. Could you please add this functionality?

**Comment by joowani** [comment]
Hi @SimonasTr,

You can update the schema by doing this:
```
my_collection.configure(schema=new_schema)
```
And remove a schema by doing this:
```
my_collection.configure(schema={})
```

I'll update the docs to make this clear. Thanks.

**Comment by joowani** [comment]
Hi @SimonasTr,

Sorry I was mistaken, the latest changes with support for schema update/delete wasn't in the latest release.
I just released 6.1.0, so please upgrade your python-arango. Thanks!
